File: boj/13311/soln13311.go
/*
// 백준
// No. 13311
// Go 1.24.8
// by "nno0obb"
*/

package main

import (
	"bufio"
	"fmt"
	"os"
)

var writer = bufio.NewWriter(os.Stdout)

func main() {
	defer writer.Flush()

	// Logic
	ans := -1 // n ≡ a-1 ≡ -1 (mod a)

	// Output
	fmt.Fprintln(writer, ans)
}

File: problem_solving/boj/13311/soln13311.py
"""
# 백준
# No. 13311
# Python 3.11.9
# by "nno0obb"
"""

from functools import reduce
from math import lcm


def main():
    # Logic
    ans = reduce(lcm, range(2, 1000 + 1), 1) - 1  # 433자리
    ans = -1  # n ≡ a-1 ≡ -1 (mod a)

    # Output
    print(ans)


if __name__ == "__main__":
    main()
